src/theory/arith/nl/nl_model.cpp

```diff
@@ -362,6 +363,11 @@ bool NlModel::addCheckModelSubstitution(TNode v, TNode s)
       return false;
     }
   }
+  auto itw = d_check_model_witnesses.find(v);
+  if (itw != d_check_model_witnesses.end())
+  {
+    // TODO(Gereon): Check whether s satisfies itw->second.
```

Comment:
We usually associate TODO with issues/wishues, for example see https://github.com/CVC4/CVC4/blob/master/src/theory/arith/nl/nl_model.h#L178.

Reply:
As per offline discussion, this case should not happen and can thus be excluded (by assertion).